model.go

```diff
@@ -1127,6 +1128,7 @@ func modelV2Fields() (schema.Fields, schema.Defaults) {
 	fields["sla"] = schema.FieldMap(
 		schema.Fields{
 			"level":       schema.String(),
+			"owner":       schema.String(),
```

Comment:
I can add a v3 if its required, but SLAs are new in Juju 2.2 and this feature is not in production yet. Do we really need to support migration of a 2.2 feature across betas?